vizro-core/src/vizro/models/_components/form/date_picker.py

```diff
@@ -0,0 +1,73 @@
+from typing import List, Literal, Optional, Union
+
+import dash_mantine_components as dmc
+from dash import html
+
+try:
+    from pydantic.v1 import Field, PrivateAttr, validator
+except ImportError:  # pragma: no cov
+    from pydantic import Field, PrivateAttr, validator
+
+
+from datetime import date
+
+from vizro.models import Action, VizroBaseModel
+from vizro.models._action._actions_chain import _action_validator_factory
+from vizro.models._components.form._form_utils import validate_max, validate_range_value
+
+
+class DatePicker(VizroBaseModel):
+    """Temporal selector `DatePicker`.
+
+    Can be provided to [`Filter`][vizro.models.Filter] or [`Parameter`][vizro.models.Parameter].
+    Based on the underlying [`dmc.DatePicker`](https://www.dash-mantine-components.com/components/datepicker).
+
+    Args:
+        type (Literal["date_picker"]): Defaults to `"date_picker"`.
+        min (Optional[date]): Start date for date picker. Defaults to `None`.
+        max (Optional[date]): End date for date picker. Defaults to `None`.
+        value (Union[List[date], date]): Default date/dates for date picker. Defaults to `None`.
+        title (str): Title to be displayed. Defaults to `""`.
+        multi (bool): Boolean flag for displaying range picker. Default to `True`.
```

Comment:
@nadijagraca @antonymilne - any preference in naming this argument? It's similar to the Dropdown but I don't find it very intuitive. 

Maybe` range=True/False?`

```suggestion
        multi (bool): Boolean flag for displaying range picker. Default to `True`.
```

Reply:
Agree - let's rename `multi` to `range`.